<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LifeWindow</class>
 <widget class="QMainWindow" name="LifeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>人生重开模拟器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 主窗口背景 */
QMainWindow {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 #1a2a6c, stop:1 #b21f1f);
}

/* 所有GroupBox样式 */
QGroupBox {
    background-color: rgba(255, 255, 255, 150);
    border: 2px solid #6a5acd;
    border-radius: 10px;
    margin-top: 10px;
    padding-top: 15px;
    font: bold 14px 'Microsoft YaHei';
    color: #333;
}

/* 按钮样式 */
QPushButton {
    background-color: #4CAF50;
    border: none;
    color: white;
    padding: 8px 16px;
    text-align: center;
    font: bold 12px;
    border-radius: 5px;
    margin: 4px;
}

QPushButton:hover {
    background-color: #45a049;
}

QPushButton:pressed {
    background-color: #3e8e41;
}

/* 标签样式 */
QLabel {
    font: 13px 'Microsoft YaHei';
    color: #333;
}

/* 文本浏览器样式 */
QTextBrowser {
    background-color: rgba(255, 255, 255, 180);
    border: 1px solid #ccc;
    border-radius: 5px;
    font: 12px 'Microsoft YaHei';
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>人物属性</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="happyLabel">
       <property name="text">
        <string>幸福：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="appearLabel">
       <property name="text">
        <string>颜值：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="wealthLabel">
       <property name="text">
        <string>财富：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="healthLabel">
       <property name="text">
        <string>健康：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ageLabel">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>年龄：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="intelLabel">
       <property name="text">
        <string>智力：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="eventGroupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>611</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>当前事件</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="eventLabel1">
       <property name="text">
        <string>这里显示事件描述：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="choicesWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="choiceButton0">
          <property name="text">
           <string>选项1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="choiceButton1">
          <property name="text">
           <string>选项2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="choiceButton2">
          <property name="text">
           <string>选项3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="choiceButton3">
          <property name="text">
           <string>选项4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>19</y>
      <width>551</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>人生轨迹</string>
    </property>
    <widget class="QTextBrowser" name="logText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>重开人生</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>choiceButton1</tabstop>
  <tabstop>choiceButton2</tabstop>
  <tabstop>choiceButton3</tabstop>
  <tabstop>choiceButton0</tabstop>
  <tabstop>logText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
